src/main/webapp/app/entities/programming-exercise/programming-exercise-detail.component.html

```diff
@@ -193,6 +193,10 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                         [participation]="programmingExercise.templateParticipation"
                     ></jhi-programming-exercise-instructions>
                 </dd>
+                <dt><span jhiTranslate="artemisApp.exercise.gradingInstructions">Grading Instructions</span></dt>
```

Comment:
Instructors can't create grading instructions for programming exercises. Either we need to allow them to do it or let's not show them

Reply:
That’s wrong! Instructors can create grading instructions. This is actively being used already!